chat-plugins/github.js

```diff
@@ -0,0 +1,94 @@
+/*
+	Github plugin by Spandamn
+	This is based on https://github.com/Ecuacion/Pokemon-Showdown-Node-Bot/blob/master/features/github/index.js
+	Yes, some of the code is copied so credits to xfix, Ecuacion, jd, and anyone who's name I forgot to put here
+*/
+'use strict';
+
+let config = {};
+
+if (!Config.github) {
+	return;
+} else if (Config.github === {}) {
+	config.port = 3420;
```

Comment:
C9 isn't meant at hosting at all, if anything, it's purely a development platform.  Actually, "hosting" is explicitly against C9's terms of service agreement.  With that in mind, I don't think we should consider C9 as a hosting platform, because it's not.